include/megaapi_impl.h

```diff
@@ -919,20 +919,20 @@ class MegaContactRequestPrivate : public MegaContactRequest
 
 #ifdef ENABLE_SYNC
 
-class MegaSyncEventPrivate: public MegaSyncEvent
+class MegaSyncEventPrivate : public MegaSyncEvent
 {
 public:
+    explicit
     MegaSyncEventPrivate(int type);
-    virtual ~MegaSyncEventPrivate();
 
-    virtual MegaSyncEvent *copy();
+    MegaSyncEvent* copy() override;
 
-    virtual int getType() const;
-    virtual const char *getPath() const;
-    virtual MegaHandle getNodeHandle() const;
-    virtual const char *getNewPath() const;
-    virtual const char* getPrevName() const;
-    virtual MegaHandle getPrevParent() const;
+    int getType() const override;
+    const char* getPath() const override;
+    MegaHandle getNodeHandle() const override;
+    const char* getNewPath() const override;
```

Comment:
No need to switch the space and * around; others are switching them to be the other way in other PRs, and it's all unnecessary - just leave those as they are, keep the PR change to a minimum.

Reply:
`const char*` is in our coding guideline, `const char *` is not so we should change it if we're making changes anyway.